tests/Assersions/AutomationAssertion.spec.ts

```diff
@@ -107,40 +107,13 @@ import path from 'path';
                 console.log('Values before clicking the arrow:', valuesDynamicTB);
                
                 //act
-                await dessertTable.locator('th:has-text("Dessert (100g)")').click();
+                await dessertTable.locator('th:has-text("Dessert (100g)")').click({delay: 3100});
```

Comment:
Please make sure next time, this gets converted to a `const`. Example:

```suggestion
                const three_seconds = 3100;
                await dessertTable.locator('th:has-text("Dessert (100g)")').click({delay: three_seconds});
```